testutils/moby-defaults.go

```diff
@@ -0,0 +1,77 @@
+package testutils
+
+import (
+	"encoding/json"
```

Comment:
nit: can we put the system imports separately at the top?

Reply:
will do, gofmt actually shuffled everything..